### ISSUE TITLE
lintPython mypy fails when importing certain dependencies

### ISSUE THREAD

**Issue by dsalaza4** [fluidattacks/makes#785]
When I do `from dateutil.relativedelta import relativedelta` in a python module and then run `lintPython` on it, I get:

```
newutils/datetime.py:8: error:(B Library stubs not installed for (B"dateutil.relativedelta"(B (or incompatible with Python 3.9)(B
newutils/datetime.py:8: note:(B Hint: (B"python3 -m pip install types-python-dateutil"(B(B
newutils/datetime.py:8: note:(B (or run (B"mypy --install-types"(B to install all missing stub packages)(B
newutils/datetime.py:8: note:(B See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports(B(B
```

If I add `types-python-dateutil: 2.8.7` to `pypi-deps.yaml` and rebuild the sources, I still get the same error.

It looks like this is being caused by having a python env for the application environment and another python env for the linter itself. More information can be found [here](https://gitlab.com/fluidattacks/product/-/issues/5297#note_775430959)

**Comment by jpverde** [comment]
@dsalaza4 Is this problem still present after the `nixpkgs` update?

**Comment by dsalaza4** [comment]
I don't really recall if I managed to fix this. I think dmurcia could have more insights regarding this error.

**Comment by kamadorueda** [comment]
It's been solved since 22.09
